pkg/monitor/monitor/template.go

```diff
@@ -477,3 +478,30 @@ func RenderPrometheusConfig(model *MonitorConfigModel) (string, error) {
 	}
 	return string(bs), nil
 }
+
+func generateAddressShardingRelabelingRules(relabelings []*config.RelabelConfig, shards uint64) []*config.RelabelConfig {
+	shardsPattern, err := config.NewRegexp("$SHARD")
```

Comment:
Is `$SHARD` expected to be a string or to be replaced by env value in the container?

Reply:
Yes,Prometheus command will sed prometheus.yml and replace `$SHARD`.